src/ocaml/driver/pparse.ml

```diff
@@ -41,26 +41,30 @@ let report_error = function
 
 external windows_merlin_system_command : string -> cwd:string -> int = "ml_merlin_system_command"
 
-let merlin_system_command =
+let commandline prog args =
+  Printf.sprintf "%s %s" prog (String.concat ~sep:" " args)
+
+let merlin_system_command ~prog ~args ~cwd =
+  let cmd = commandline prog args in
   if Sys.win32 then
```

Comment:
We probably want to move that in the (appropriately renamed) `System_command.unix` function so that users of the lib can also redefine the windows spawning.